src/screens/VisualTests/BuildResults.tsx

```diff
@@ -74,6 +74,14 @@ export const BuildResults = ({
       "testsForStory" in storyBuild ? storyBuild.testsForStory.nodes : []
     ),
   ];
+  const nextStoryTests = [
+    ...getFragment(
+      FragmentNextStoryTestFields,
+      "testsForStory" in nextBuild ? nextBuild.testsForStory.nodes : []
+    ),
+  ];
+  const isStoryOutdated =
+    !isReviewable && nextStoryTests.every(({ status }) => status !== TestStatus.InProgress);
```

Comment:
Can't this be achieved by looking at `nextBuild.completedAt` or `nextBuild.result`?

Reply:
We aren't waiting for the whole build to finish, just all the tests for the current story.